templates/openwisp2/settings.py

```diff
@@ -359,6 +375,20 @@
 RAVEN_CONFIG = {{ openwisp2_sentry|to_nice_json }}
 {% endif %}
 
+{% if openwisp2_monitoring %}
+TIMESERIES_DATABASE = {
+    'BACKEND': '{{ openwisp2_timeseries_database.backend }}',
+    'USER': '{{ openwisp2_timeseries_database.user }}',
+    'PASSWORD': '{{ openwisp2_timeseries_database.password }}',
+    'NAME': '{{ openwisp2_timeseries_database.name }}',
+    'HOST': '{{ influxdb_http_ip }}',
+    'PORT': '{{ influxdb_http_port }}',
+}
+
+INSTALLED_APPS.append('djcelery_email')
+EMAIL_BACKEND = '{{ openwisp2_email_backend }}'
```

Comment:
It should not be in this block @Purhan. Since we added a dedicated setting to allow configuring email settings, it should be universal. Users should be able to configure it without using openwisp-monitoring. 

Do you understand what I mean here?